lib/src/boom_menu.dart

```diff
@@ -79,6 +85,11 @@ class BoomMenu extends StatefulWidget {
     this.onClose,
     this.overlayVisible = false,
     this.fabMenuBorder = const CircleBorder(),
+    this.fabAlignment = Alignment.centerRight,
+    this.fabPaddingRight = 0,
+    this.fabPaddingLeft = 0,
+    this.fabPaddingBottom = 0,
+    this.fabPaddingTop = 0,
```

Comment:
this.fabPaddingTop = 0,
Remove this Line(92). it's make a Bottom overflowed error.